spyder/widgets/sourcecode/base.py

```diff
@@ -34,7 +33,8 @@
 from spyder.widgets.calltip import CallTipWidget
 from spyder.widgets.mixins import BaseEditMixin
 from spyder.widgets.sourcecode.terminal import ANSIEscapeCodeHandler
-
+from spyder.widgets.sourcecode.api.decoration import TextDecoration
+from spyder.widgets.sourcecode.utils.decoration import TextDecorationsManager
 
 def insert_text_to(cursor, text, fmt):
     """Helper to print text, taking into account backspaces"""

```

Comment:
period at the end of the docstring ;-)

Reply:
I didn't add this docstring 🙈  (I'll improve it with some clarifications of the  new behavior and fix that style error. )

Reply:
I know, but its a ood opportunity to clean it up a bit :-)

Reply:
I improved other docstrings, but I didn't modify this function at all, fixing that docstring will add noise to the PR